From f65168f82d6b7795fdd5f2efe3ad10830a87dcbb Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Tue, 29 Apr 2014 15:05:05 +0200
Subject: [PATCH] lsusb-t: don't segfault when usbbuslist is empty

---
 lsusb-t.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lsusb-t.c b/lsusb-t.c
index 0f2c5bf..a849d4c 100644
--- a/lsusb-t.c
+++ b/lsusb-t.c
@@ -609,7 +609,7 @@ static void sort_busses(void)
 		p = usbbuslist;
 		pp = &usbbuslist;
 		swapped = 0;
-		while (p->next) {
+		while (p && p->next) {
 			if (p->busnum < p->next->busnum) {
 				t = p->next;
 				p->next = t->next;
-- 
1.8.3.1

