From abd6f3fc71ad941f3dd5fb68a0dfca4e01720dd9 Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Wed, 2 Dec 2015 09:51:24 +0100
Subject: [PATCH 01/16] change path to hwdata

---
 Makefile.am | 25 +------------------------
 lsusb.py    |  2 +-
 2 files changed, 2 insertions(+), 25 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index aa3644a..559f3ed 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -4,15 +4,9 @@ SUBDIRS = \
 AM_LDFLAGS = \
 	-Wl,--as-needed
 
-data_DATA = \
-	usb.ids
-
 bin_PROGRAMS = \
 	lsusb
 
-sbin_SCRIPTS = \
-	update-usbids.sh
-
 bin_SCRIPTS = \
 	usb-devices \
 	lsusb.py
@@ -27,37 +21,22 @@ lsusb_SOURCES = \
 
 lsusb_CPPFLAGS = \
 	$(AM_CPPFLAGS) $(LIBUSB_CFLAGS) \
-	-DDATADIR=\"$(datadir)\"
+	-DDATADIR=\"$(datadir)/hwdata\"
 
 lsusb_LDADD = \
 	$(LIBUSB_LIBS)
 
-if HAVE_ZLIB
-data_DATA += usb.ids.gz
-lsusb_CPPFLAGS += -DHAVE_LIBZ
-lsusb_LDADD += -lz
-endif
-
 man_MANS = \
 	lsusb.8	\
 	usb-devices.1
 
 EXTRA_DIST = \
-	usb.ids \
-	update-usbids.sh.in \
 	lsusb.8.in \
 	usb-devices.1.in \
 	usb-devices \
 	lsusb.py \
 	usbutils.pc.in
 
-usb.ids.gz: $(srcdir)/usb.ids
-	gzip -c -9 $< > $@
-
-update-usbids.sh: $(srcdir)/update-usbids.sh.in
-	sed 's|@usbids@|$(datadir)/usb.ids|g' $< >$@
-	chmod 755 $@
-
 lsusb.8: $(srcdir)/lsusb.8.in
 	sed 's|VERSION|$(VERSION)|g;s|@usbids@|$(datadir)/usb.ids|g' $< >$@
 
@@ -71,10 +50,8 @@ usbutils.pc: $(srcdir)/usbutils.pc.in
 	sed 's|@usbids@|$(datadir)/usb.ids|g;s|@VERSION[@]|$(VERSION)|g' $< >$@
 
 DISTCLEANFILES = \
-	usb.ids.gz \
 	lsusb.8 \
 	usb-devices.1 \
-	update-usbids.sh \
 	usbutils.pc
 
 distclean-local:
diff --git a/lsusb.py b/lsusb.py
index 6703414..a743db0 100755
--- a/lsusb.py
+++ b/lsusb.py
@@ -16,7 +16,7 @@ nohub = False
 warnsort = False
 
 prefix = "/sys/bus/usb/devices/"
-usbids = "/usr/share/usb.ids"
+usbids = "/usr/share/hwdata/usb.ids"
 
 esc = chr(27)
 norm = esc + "[0;0m"
-- 
2.4.3

