From 222110010f7f359191060a835185420968100912 Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Wed, 2 Dec 2015 09:52:11 +0100
Subject: [PATCH 02/16] invalid config descriptors

---
 lsusb.c | 23 +++++++++++++++++------
 1 file changed, 17 insertions(+), 6 deletions(-)

diff --git a/lsusb.c b/lsusb.c
index 6ba2288..25dbc51 100644
--- a/lsusb.c
+++ b/lsusb.c
@@ -3800,14 +3800,25 @@ static void dumpdev(libusb_device *dev)
 	if (desc.bNumConfigurations) {
 		struct libusb_config_descriptor *config;
 
-		libusb_get_config_descriptor(dev, 0, &config);
-		otg = do_otg(config) || otg;
-		libusb_free_config_descriptor(config);
+		ret = libusb_get_config_descriptor(dev, 0, &config);
+		if (ret) {
+			fprintf(stderr, "Couldn't get configuration descriptor 0, "
+					"some information will be missing\n");
+		} else {
+			otg = do_otg(config) || otg;
+			libusb_free_config_descriptor(config);
+		}
 
 		for (i = 0; i < desc.bNumConfigurations; ++i) {
-			libusb_get_config_descriptor(dev, i, &config);
-			dump_config(udev, config);
-			libusb_free_config_descriptor(config);
+			ret = libusb_get_config_descriptor(dev, i, &config);
+			if (ret) {
+				fprintf(stderr, "Couldn't get configuration "
+						"descriptor %d, some information will "
+						"be missing\n", i);
+			} else {
+				dump_config(udev, config);
+				libusb_free_config_descriptor(config);
+			}
 		}
 	}
 	if (!udev)
-- 
2.4.3

